docs/proposals/operator-hub.md

```diff
@@ -0,0 +1,197 @@
+csvs` doesn’t work since there’s no API resources with that name in Kubernetes.
+
+One needs to manually install OLM from its
+[repo](https://github.com/operator-framework/operator-lifecycle-manager/tree/00eab85df0fd570891754cc743bc3d5831e9dd62/deploy/upstream/quickstart)
+to make ClusterServiceVersion objects available in Kubernetes.
+
+## Link components with operands
+[GitHub issue](https://github.com/openshift/odo/issues/2463)
+
+In its current form, linking a component with another component/service
+involves making the Secrets (environment variables) of the component/service
+being linked available to the component it is linked with. For example, if
+`component A` needs to be connected to `service A`  then the environment variables
+of `service A` will be made available in `component A`.
+
+Although Service Binding Operator could provide an easy mechanism to link the
+components with operands created from an operator, we intend to explore linking
+the two without adding the dependency on this operator. This should help us be
+agnostic of backend cluster being OpenShift or Kubernetes.
+
+## Listing the services
+[GitHub issue](https://github.com/openshift/odo/issues/2479)
+
+At the moment, we use `odo service list --app <app-name> --project
+<project-name>` to list the services in a given application and project. We
+could augment this to have the information about the services created from the
+Operators. To make a distinction between the source of the services in the
+cluster, we could add a column to the output just like we plan to do for
+listing available/installed operators via `odo catalog list services`.
+
+```sh
+$ odo services list
+NAME                          PLANS                    PROVISIONER
+mongodb-enterprise.v1.2.4     prod                     OperatorHub
+mariadb-operator-0.1.3-6      ephemeral,persistent     ServiceCatalog
+```
+
+## Describe deployed services
+[GitHub issue](https://github.com/openshift/odo/issues/2480)
+
+At the moment there’s no command in odo that describes a deployed service. We
+have  a command to describe a service in the Service Catalog: `odo catalog
+describe service <service-name>`. But there’s no way to describe a deployed
+service. We can only list the deployed services with `odo service list`.
+
+If we intend to add a command to describe the deployed service(s), we need to:
+1. Check with current users (CLI users, IDE plugin teams, GSS) if this is
+something that would help them.
+2. Define what “describing a deployed service” exactly means
+3. Decide what should be shown to the user as the output on CLI
+4. Implement the command for both ServiceCatalog and OperatorHub
+
+## Delete the service
+[GitHub issue](https://github.com/openshift/odo/issues/2481)
+
+At the moment, we use `odo service delete <service-name>` to delete the service
+deployed in OpenShift cluster. We should be able to delete the service
+deployed using Operator Hub with the same command.
+Testing on Kubernetes
```

Comment:
Is this supposed to be under the `Delete the service` headline?
It looks like ti might be copy/paste error.